pkg/mock/v1beta1/api/earlystopping.go

```diff
@@ -6,10 +6,11 @@ package mock
 
 import (
 	context "context"
+	reflect "reflect"
```

Comment:
We might need to define specific version for `mockgen` to have same generation files.
Do we need these changes in the PR ?